database/factories/LanguageFactory.php

```diff
@@ -20,8 +20,7 @@
 
     return [
         'name' => $name,
-        'lang' => ucfirst($name),
-        'iso' => Str::slug($name),
+        'code' => Str::slug($name),
```

Comment:
Considering this will normally be a code, should we be basing it off of the name for factories?